mbuild/compound.py

```diff
@@ -190,6 +190,24 @@ def periodicity(self):
     def periodicity(self, periods):
         self._periodicity = np.array(periods)
 
+    def view_hierarchy(self, show_ports=False):
+        """View a Compounds hierarchy of compounds as a chart. """
+        import networkx as nx
```

Comment:
Let's add a try/except around these imports since they're really optional and provide a more descriptive error message.